blockchain/types/accountkey/account_key.go

```diff
@@ -108,16 +109,16 @@ func NewAccountKey(t AccountKeyType) (AccountKey, error) {
 	return nil, errUndefinedAccountKeyType
 }
 
-func ValidateAccountKey(from common.Address, accKey AccountKey, pubkeys []*ecdsa.PublicKey, roleType RoleType) error {
+func ValidateAccountKey(from common.Address, accKey AccountKey, recoveredKeys []*ecdsa.PublicKey, roleType RoleType) error {
 	// Special treatment for AccountKeyLegacy.
 	if accKey.Type().IsLegacyAccountKey() {
```

Comment:
@aidan-kwon If we implement validation logic in AccountKeyLegacy, it seems that this if statement is not necessary.

Reply:
You're right. I removed it. Thanks!